platform/backend/bridge/src/integrationTest/kotlin/io/hamal/bridge/http/controller/code/CodeGetControllerTest.kt

```diff
@@ -112,13 +119,45 @@ internal class CodeGetControllerTest : CodeBaseControllerTest() {
 
     }
 
+    @Disabled
```

Comment:
Why is this test disabled?

Reply:
Didn´t throw exception I expected. Removed as in /api/../ CodeGetControllerTest.